demo.html

```diff
@@ -160,133 +152,118 @@ <h1 class="section-title">Body copy</h1>
-
+      </header>
+      <header class="sections" id="address">
+          <h1 class="section-title">Addresses</h1>
         <div class="breakdown">
-          <p>Make a paragraph stand out by adding <code class = "class-name">.vast-lead</code>.
+          <p>Present contact information in our default style by using the <code class="element-in-text">&lt;address&gt;</code> tag. Preserve your formatting by ending all lines with <code class="element-in-text">&lt;br&gt;</code>.
           </p>
         </div>
-
         <div class="flex-container sample-text">
-          <p class="vast-lead">Post-ironic shoreditch chambray whatever fixie, butcher crucifix cronut migas vexillologist microdosing flannel beard. Vinyl cold-pressed echo park pickled, venmo paleo kogi whatever. Organic pitchfork drinking vinegar, microdosing pinterest stumptown live-edge. Chambray green juice fingerstache drinking vinegar. Waistcoat truffaut mlkshk scenester, fingerstache vape yuccie single-origin coffee flannel quinoa man braid polaroid four dollar toast deep v. Glossier cardigan franzen slow-carb kombucha banh mi. Vexillologist mixtape stumptown small batch vinyl, intelligentsia snackwave synth thundercats af unicorn fap ramps seitan.
-          </p>
-        </div>
-
-        <div class="sample-code">
-          <code>
-            <p>
-              <span class="element-color">&lt;p</span>
-              <span class="class">class=</span><!--
-           --><span class="class-name">"vast-lead"</span><!--
-           --><span class="element-color">&gt;</span><!--
-           -->...<!--
-           --><span class="element-color">&lt;/p&gt;</span>
-            </p>
-          </code>
-        </div>
-    </header>
-
-    <header class="sections" id="address">
-      <h1 class="section-title">Addresses</h1>
-
-      <div class="breakdown">
-        <p>Present contact information in our default style by using the <code class="element-in-text">&lt;address&gt;</code> tag. Preserve your formatting by ending all lines with <code class="element-in-text">&lt;br&gt;</code>.
-        </p>
-      </div>
-
-      <div class="flex-container sample-text">
           <p>
             <address>
               <strong>Learners Guild</strong><br>
               492 Ninth Street (Garden Level)<br>
               Oakland, CA 94607<br>
               <abbr title="Phone">P:</abbr> (123) 456-7890
             </address>
-
             <address>
               <strong>Full Name</strong><br>
               <a href="mailto:#">first.last@example.com</a>
             </address>
           </p>
         </div>
-
-      <div class="sample-code">
-          <code>
+        <div class="sample-code">
+            <code>
+              <p>
+                <span class="element-color">&lt;address&gt;</span><br>
+                <span class="tab element-color">&lt;strong&gt;</span><!--
+             -->Learners Guild<!--
+             --><span class="element-color">&lt;/strong&gt;&lt;br&gt;</span><br>
```

Comment:
Completely inconsequential, but the comments between spans are a bit baffling to me 0.o